contracts/LifCrowdsale.sol

```diff
@@ -310,19 +358,13 @@ contract LifCrowdsale is Ownable, Pausable {
     require(!isFinalized);
     require(hasEnded());
 
-    // TODO: transfer an extra 25% of tokens to the foundation, for the team
-    // TODO: transfer 13% to founders with a vesting mechanism?
-
     // foward founds and unpause token only if minCap is reached
     if (funded()) {
 
-      // finish the minting of the token, unpause it and transfer the ownership to the foundation
-      token.finishMinting();
-      token.unpause();
-      token.transferOwnership(owner);
-
       forwardFunds();
 
+      finishDistribution();
```

Comment:
I think `finishDistribution` should be part of or called from `forwardFunds`. There we know the length of the MVM so we could pass it as a parameter to `finishDistribution`, instead of having it to ask it to the MVM. That way we decouple all that part from the MVM

Another approach would be to extract small functions for each part of the `finishDistribution` and call them from `forwardFunds`. finishDistribution doesn't feel like a very good name, given we already have `finalize` and `forwardFunds`, it's not clear what belongs to where.

Reply:
I thought about that but one thing is to forward the ethers raised and another its to distribute the 25% of the tokens, for me they are two separated things, and the function name its also ok for me, I mean..its what it does..it finish the distribution of the tokens. So at the end of the crowdsale it does two things: forward the ethers raised to the foundation and MVM, and it finish the distribution of the token too.

Reply:
I think the name it's too abstract. And it's not even symmetric to the other stuff that is being done here. Like if it was `startDistribution` and `finishDistribution`, at least there would be some symmetry, even though I wouldn't say I like those names :). 

This is about token distribution, right? Maybe it can be called `mintAndDistributeExtraTokens`, but it shouldn't take care of the unpause/transferOwnership stuff. Also I insist that it shouldn't take the vesting length from the MVM, it should receive it as argument

Reply:
On changing the name I agree, but not on executing the function inside forwardFunds, for me they are two separate things, and in the function we are taking in count if the MVM was created we create the vestingSchedule with the right amount of periods.

Reply:
ok